specification/riskiq/data-plane/Microsoft.Easm/preview/2022-09-01-preview/easm.json

```diff
@@ -0,0 +1,5550 @@
+        },
+        "cweId": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "DataConnectionPageResponse": {
+      "properties": {
+        "content": {
+          "description": "The items in the current page of results.",
+          "items": {
+            "$ref": "#/definitions/DataConnectionResponse"
+          },
+          "type": "array"
+        },
+        "first": {
+          "description": "Whether or not this is the first page of results.",
+          "type": "boolean"
+        },
+        "last": {
+          "description": "Whether or not this is the last page of results.",
+          "type": "boolean"
+        },
+        "nextLink": {
+          "description": "The link to access the next page of results.  Not set if at the end of the result set.",
+          "type": "string"
+        },
+        "number": {
+          "description": "The page number requested.",
+          "format": "int32",
+          "type": "integer"
+        },
+        "numberOfElements": {
+          "description": "The number of items in the current page of results.",
+          "format": "int32",
+          "type": "integer"
+        },
+        "size": {
+          "description": "The page size requested.",
+          "format": "int32",
+          "type": "integer"
+        },
+        "totalElements": {
+          "description": "The total number of items available in the full result set.",
+          "format": "int64",
+          "type": "integer"
+        },
+        "totalPages": {
+          "description": "The total number of pages available in the full result set.",
+          "format": "int32",
+          "type": "integer"
+        }
+      },
+      "type": "object"
+    },
+    "DataConnectionRequest": {
+      "properties": {
+        "connectionString": {
+          "description": "The connection string for the data connection.",
+          "type": "string"
+        },
+        "kind": {
+          "description": "The type of the data connection.",
+          "enum": [
+            "sentinel"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "dataConnectionRequestKind"
+          }
+        },
+        "options": {
+          "description": "Any connection specific options.",
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "DataConnectionResponse": {
+      "properties": {
+        "connectionString": {
+          "description": "The connection string for the data connection.",
+          "type": "string"
+        },
+        "createdDate": {
+          "description": "The date the data connection was created.",
+          "format": "date-time",
+          "type": "string"
+        },
+        "displayName": {
+          "type": "string"
+        },
+        "id": {
+          "type": "string"
+        },
+        "kind": {
```

Comment:
Same here.

Reply:
I don't see anything "deriving" from this, so presumably it's not a discriminator. Please be sure to reply to comments before resolving.